README.md

```diff
@@ -13,22 +13,22 @@ Strapi is an open source Node.js Headless CMS to easily build customisable APIs
 ---
 
 <!-- vscode-markdown-toc -->
-* [Documentation, Tutorials, and Meetups](#DocumentationTutorialsandMeetups)
-* [One-Click & Deployment](#One-ClickDeployment)
-* [Starters & Examples](#StartersExamples)
-	* [React](#React)
-	* [Gatsby](#Gatsby)
-	* [Vue.js](#Vue.js)
-	* [Next.js](#Next.js)
-	* [Nuxt.js](#Nuxt.js)
-	* [Angular](#Angular)
-* [Plugin & Providers](#PluginProviders)
-	* [Official Plugins](#OfficialPlugins)
-	* [Community Plugins](#CommunityPlugins)
-	* [Email Providers](#EmailProviders)
-	* [Upload Providers](#UploadProviders)
-* [Ecosystem](#Ecosystem)
-	* [SDKs](#SDKs)
+
```

Comment:
@zeybek can you revert the changes to the TOC. The extension for VSCode used to generate it is:
https://github.com/joffreykern/vscode-markdown-toc
https://marketplace.visualstudio.com/items?itemName=joffreykern.markdown-toc

Reply:
Fixed, sorry about that